AmazonChimeSDK/AmazonChimeSDK/internal/audio/DefaultAudioClientObserver.swift

```diff
@@ -197,6 +198,60 @@ class DefaultAudioClientObserver: NSObject, AudioClientDelegate {
         }
     }
 
+    public func transcriptEventsReceived(_ events: [Any]?) {
+        if events == nil {
+            return
+        }
+
+        guard let internalEvents = events as? [TranscriptEventInternal] else {
+            return
+        }
+
+        internalEvents.forEach { rawEvent in
+            ObserverUtils.forEach(observers: transcriptEventObservers) { (observer: TranscriptEventObserver) in
+                if let rawStatus = rawEvent as? TranscriptionStatusInternal {
```

Comment:
Not a swift expert, is this process being done for each `transcriptEventObserver`?

Reply:
Good catch! Yes this is not optimized and will be processed for each observer. Will fix.